python/lsst/afw/detection/peak.cc

```diff
@@ -111,11 +111,9 @@ PYBIND11_PLUGIN(_peak) {
     table::pybind11::declareCatalog(clsPeakCatalog);
 
     clsPeakRecord.attr("Table") = clsPeakTable;
-    clsPeakRecord.attr("ColumnView") = clsPeakColumnView;
```

Comment:
I added it (but not for `Catalog`) in one commit and removed it in the last commit because it's just clutter. I can leave it in if desired. I don't think it is useful in Python and adds clutter.